Sources/BraintreeThreeDSecure/BTThreeDSecureV2Provider.m

```diff
@@ -48,14 +48,16 @@ + (instancetype)initializeProviderWithConfiguration:(BTConfiguration *)configura
                                          completion:(BTThreeDSecureV2ProviderInitializeCompletionHandler)completionHandler {
     BTThreeDSecureV2Provider *instance = [self new];
     instance.apiClient = apiClient;
-    instance.cardinalSession = [CardinalSession new];
-    CardinalSessionConfiguration *cardinalConfiguration = [CardinalSessionConfiguration new];
-    if (request.uiCustomization) {
-        cardinalConfiguration.uiCustomization = request.uiCustomization;
-    }
-    CardinalSessionEnvironment cardinalEnvironment = CardinalSessionEnvironmentStaging;
+
+    instance.cardinalSession = (id<BTProxyCardinalSession>)[NSClassFromString(@"CardinalSession") new];
+    id<BTProxyCardinalSessionConfiguration> cardinalConfiguration = (id<BTProxyCardinalSessionConfiguration>)[NSClassFromString(@"CardinalSessionConfiguration") new];
+    // TODO: Enable UI customization
```

Comment:
I take it this TODO will be addressed in a later PR?

Reply:
Yes! We had to take a slightly different approach on the UI customization, so we'll be opening a separate PR for it shortly.

Reply:
https://github.com/braintree/braintree_ios/pull/569